gossipd/routing.c

```diff
@@ -1501,6 +1502,20 @@ bool routing_add_channel_update(struct routing_state *rstate,
 			hc->bcast.index = index;
 			hc->rgraph.index = index;
 		}
+		if (!peer)
+			return true;
```

Comment:
mh from a dummy person in gossipd, why if the peer is `NULL` we add gossip update?